### ISSUE TITLE
Be more helpful when an incompatible Pod is used

### ISSUE THREAD

**Issue by neonichu** [neonichu/Rome#15]
Currently known to be incompatible:

- anything which only consists of prebuilt binaries (we could copy frameworks over if they exist, but also consider things like Helpshift, which use `vendored_library` or in that case `library`)

**Comment by segiddins** [comment]
Does https://github.com/neonichu/Rome/pull/18 help this at all?

**Comment by neonichu** [comment]
Not really, IMO we should just copy vendored frameworks over, CP doesn't need to touch them, but for vendored libraries, there's no good solution.

**Comment by segiddins** [comment]
I think generating an xcconfig would go a long way? On the other hand, the user can still just drag vendored libs

**Comment by neonichu** [comment]
Yah, but they also have to drag headers, so XCConfig sounds good
